src/goto-programs/lazy_goto_model.cpp

```diff
@@ -207,6 +208,24 @@ void lazy_goto_modelt::load_all_functions() const
 
 bool lazy_goto_modelt::freeze()
 {
+  messaget msg(language_files.get_message_handler());
+  journalling_symbol_tablet symbol_table=
+    journalling_symbol_tablet::wrap(this->symbol_table);
+  if(language_files.final(symbol_table))
+  {
+    msg.error() << "FINAL STAGE CONVERSION ERROR" << messaget::eom;
+    return false;
+  }
+  for(const irep_idt &updated_symbol_id : symbol_table.get_updated())
+  {
+    if(is_function_symbol(*symbol_table.lookup(updated_symbol_id)))
+    {
+      // Re-convert any that already exist
+      goto_functions.unload(updated_symbol_id);
+      goto_functions.at(updated_symbol_id);
```

Comment:
I'm not sure I enjoy the implicitness here.

Reply:
I see what you mean. I think in any "real" policy (not just "load all") the body of this function would be explored, making it more explicit.

Reply:
Would you mind adding some API (unless it exists?) so that you can do `goto_functions.convert(updated_symbol_id);` instead of using `at`? I'm perfectly fine with that function just wrapping a call to `at`, but this code is not very descriptive (as witnessed by the fact that you felt compelled to add a comment).